### DISCUSSION TITLE
How to resolve vertical "sticking" across rows in project cards

### DISCUSSION THREAD

**Discussion by MakoStrwlkr** [alshedivat/al-folio#2773] [Q&A]
Hi! Thanks for this template, it has been really smooth using it so far, and easily customisable.

I was recently adding some more of my projects to my website, and noticed that, when I have multiple rows of projects in the same category, the respective project cards are too close to each other across the rows. For instance, I included a screenshot of my problem below.
![image](https://github.com/user-attachments/assets/d5f88f2e-ba0b-4f38-b327-43a45e678c11)

I have tried scouting the code responsible for displaying the project cards, but can't seem to figure out what could be causing this issue. I'm especially confused because I don't think I have ever touched any of the styling or layout files in the template.

**Comment by riasat-sheikh** [comment]
Are you using the latest update? Even in the [al-folio's original website](https://alshedivat.github.io/al-folio/projects/) I can't see this problem!
